### DISCUSSION TITLE
Eth Token

### DISCUSSION THREAD

**Discussion by Mertz22** [community/community#54452] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![439FFC89-1684-4664-BC5A-5B04109AEAD3](https://user-images.githubusercontent.com/99062806/236253929-018bd6fd-aa17-433a-a448-38b02f617ff5.png)

**Comment by Mertz22** [comment]
I got some wallet with Ethereum bnb and polygons fund but i wasn’t able to pull it out who can help

**Comment by samus-aran** [comment, marked as answer]
Hi @Mertz22 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
